components/floatinglabel/overview.md

```diff
@@ -0,0 +1,106 @@
+
+
+## Benefits
+
+The Telerik FloatingLabel enhances HTML `label` functionality in the following ways:
+
+* built-in animations
+* integration with form validation
+* integration with the `placeholder` attribute
+* association with non-form elements - for example, the TelerikDropDownList component renders as a `<span>`
+
+
+## Creating Blazor FloatingLabel
+
+1. Use the `TelerikFloatingLabel` tag.
+1. Set the `Text` parameter of the floating label.
+1. Place a [compatible Telerik component](#compatibility) inside the floating label.
+1. Set an `Id` to the Telerik component. Otherwise the floating label will not render a `for` attribute and label clicks will not focus the associated component.
+
+>caption How to use a Floating Label
+
+````CSHTML
+<TelerikFloatingLabel Text="Your Name">
+    <TelerikTextBox Id="name" @bind-Value="@Name" />
+</TelerikFloatingLabel>
+
+@code {
+    string Name { get; set; }
+}
+````
+
+
+## Compatibility
+
+The FloatingLabel integrates with focusable Telerik Blazor components:
+
+* AutoComplete
+* ComboBox
+* DateInput
+* DatePicker
+* DateTimePicker
+* DropDownList
+* MaskedTextBox
+* MultiSelect
+* NumericTextBox
+* TextArea
+* TextBox
+* TimePicker
+
+The FloatingLabel does not support third-party components and generic HTML inputs.
+
+
+## Forms and Validation
+
+The FloatingLabel can change its styles, based on validation state. See [Validation]({%slug floatinglabel-validation%}) for details and example.
+
+
+## Placeholder Behavior
+
+Labels and placeholders serve similar purpose for the user experience.
+
+If a Telerik component has both a `Placeholder` and a floating label, the behavior is:
+
+* When the floating label is **over the component**, then the placeholder is **not rendered**.
+* When the floating label is **outside the component** and the focused component has no value, the placeholder **is visible**.
+
+
+## FloatingLabel Parameters
+
+The following table lists the FloatingLabel parameters. Also check the [FloatingLabel API Reference](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Components.TelerikFloatingLabel) for a full list of properties, methods and events.
+
+<style>
+    article style + table {
+        table-layout: auto;
+        word-break: normal;
+    }
+</style>
+
+| Attribute | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | Renders additional CSS class to the `span.k-floating-label-container` element, which holds the `<label>` child element. Use `Class` to apply custom styles or [override the theme]({%slug themes-override%}). |
+| `Id` | `string` | Renders an `id` attribute to the  `label.k-label` element. To improve accessibility, set it together with the `AriaLabelledBy` parameter of the associated Telerik component. |
```

Comment:
I would suggest something like "renders an id attribute to the label, so that it can match the `AriaLabelledBy` parameter of the associated Telerik component.